.github/workflows/backend-test.yml

```diff
@@ -13,9 +13,27 @@ jobs:
     strategy:
       matrix:
         java-version: [ 17 ]
+    # Service containers to run with `runner-job`
+    services:
+      packit-db:
+        image: mrcide/packit-db:${{ github.head_ref || github.ref_name }}
+        # Set health checks to wait until postgres has started and add container name
+        options: >-
+          --health-cmd pg_isready
```

Comment:
this is cool, so much easier than managing all this ourselves on buildkite!

Reply:
It's awesome. We can do the same in docker-compose file. I think we should properly have a ticket for dockerising the apps.